src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs

```diff
@@ -156,13 +156,13 @@ public ProcessResult SendPdfEmail(string html, string emailAddress, int orderNum
                     pdf.Result,
                     $"LinnPurchaseOrder{orderNumber}");
 
+            order.SentByMethod = "EMAIL";
+
+            //// todo When remove mini orders: stop setting sentbymethod mini order and switch to below
```

Comment:
leaving the to do in?

Reply:
yeah, can't remove mini order for a while! Unless you mean the actual todo, in which case yeah happy to leave it here just so it's super obvious, the rest are in their own methods but haven't done it here for updating 1 thing. Could but doesn't seem worth it. Although I changed the code so it doesn't actually make sense will update it haha